pkg/karpenter/karpenter.go

```diff
@@ -12,28 +12,37 @@ import (
 const (
 	// DefaultKarpenterNamespace default namespace for Karpenter
 	DefaultKarpenterNamespace = "karpenter"
+	// DefaultKarpenterServiceAccountName is the name of the service account which is needed for Karpenter
+	DefaultKarpenterServiceAccountName = "karpenter"
```

Comment:
these 2 could be renamed to `karpenter.DefaultNamespace` & `karpenter.DefaultServiceAccountName`